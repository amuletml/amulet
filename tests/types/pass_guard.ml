external val (>) : int -> int -> bool = ""

let foo = function
  | x when x > 2 -> x
  | _ -> 2
