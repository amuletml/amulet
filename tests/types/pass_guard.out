> : int -> int -> bool
foo : int -> int
